junit5/README.md

```diff
@@ -382,18 +383,21 @@ class MyTest {
     }
 }
 ```
+
 ## WeldJunit5AutoExtension
 
 To use this approach, annotate your test class with `ExtendWith(WeldJunit5AutoExtension.class)` or just `@EnableAutoWeld`.
 By default, the extension will:
 
-* Inspect your test class and try to figure out what beans classes it needs based on injection points (field and parameter injection both work)
+* Inspect your test class and try to figure out what bean classes it needs based on injection points (field and parameter injection both work)
   * This is done by finding classes and verifying if they have [bean defining annotation](http://docs.jboss.org/cdi/spec/2.0/cdi-spec.html#bean_defining_annotations) so make sure they do
 * Add those classes to Weld container
 * Process additional annotations on test class
   * `@AddPackages`, `@AddExtensions`, `@ActivateScopes`, ...
+* Add the test classes to Weld container with the instantiation replaced with the test instance provided by JUnit
```

Comment:
Same as the comment above.

Reply:
Sorry for asking this but what other comment above do you refer to?

Reply:
No need to be sorry, the ordering in review is apparently different from what I saw when I was writing it. I don't really know what was this supposed to be linked to anymore, but I see no issue with it now.